libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -278,142 +299,149 @@ export class XCCDFResultsMapper extends BaseConverter {
         status: 'loaded',
         controls: [
           {
-            path: 'Benchmark.Group',
+            path: 'Benchmark',
+            pathTransform: getRulesInBenchmark,
             key: 'id',
             tags: {
               cci: {
-                path: 'Rule.ident',
+                path: ['ident', 'reference'],
                 transformer: extractCci
               },
               nist: {
-                path: 'Rule.ident',
+                path: ['ident', 'reference'],
                 transformer: nistTag
               },
-              severity: {path: 'Rule.severity'},
+              severity: {path: 'severity'},
               description: {
```

Comment:
![image](https://github.com/mitre/heimdall2/assets/2015821/dfd9948a-74bf-49d8-9086-0263f9b34905)

requires more work to handle the embedded html tags in there - "theso" came from "place a file in the <html blob> so that it" but the html blob and surrounding whitespace was removed for some reason

Reply:
This problem appears to be caused by shortcoming the XML parser used by this project: https://github.com/NaturalIntelligence/fast-xml-parser/issues/558

I don't know what, if anything, I can do about this one... any suggestions?

Reply:
Yeah, we rewrite it using JSONIX which hopefully just leaves random embedded blocks as they are.  So long as the with-raw option is provided, this isn't a blocker cause the user could at the end of the day retrieve this information.  I'm not gonna make this a blocker.